assets/js/copyable.js

```diff
@@ -0,0 +1,29 @@
+async function copyText(e) {
+    const btn = e.target;
+    const textNode = btn.previousElementSibling;
+    try {
+        await navigator.clipboard.writeText(textNode.textContent);
+        btn.textContent = '✅';
+    } catch (err) {
+        alert(`Error copying text to clipboard: ${err.message}\n\nPlease copy manually instead.`);
+        const selection = window.getSelection();
+        const range = document.createRange();
+        range.selectNodeContents(textNode);
+        selection.removeAllRanges();
+        selection.addRange(range);
+        btn.textContent = '❌';
+    }
+    setTimeout(() => {
+        btn.textContent = '📋';
+    }, 1000);
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+    for (const el of document.querySelectorAll('.copyable')) {
+        const btn = document.createElement('button');
+        btn.title = 'Copy to clipboard';
+        btn.textContent = '📋';
```

Comment:
I prefer to use a "wireframe" type icon such as https://fonts.google.com/icons?selected=Material%20Icons%3Acontent_copy%3A

This comment applies to all the icons (X and Checkmark)

Reply:
In my first attempt I was using an icon like that, but over time I have grown to prefer the icons that I use now. They feel more actionable to me.

Reply:
I'll give it a look, but they seem out of place compared to the rest of our page styling.

Reply:
Oh.. I saw your change too late. Anyway, here's what the fork awesome icons look like:

<img width="62" alt="Screenshot 2023-09-16 at 13 24 13" src="https://github.com/keepassxreboot/keepassxc-org/assets/1991151/9a9e6f95-0035-4887-b2eb-99427835f5ea">

<img width="60" alt="Screenshot 2023-09-16 at 13 24 30" src="https://github.com/keepassxreboot/keepassxc-org/assets/1991151/356062f4-74ba-4a48-8d82-0f83337f3eb3">

I removed the error icon since it didn't look that good. I don't think an icon is strictly necessary in that case since there's the alert dialog.

I can remove my last commit if you prefer the unicode characters instead.